client/js/admin.js

```diff
@@ -1,49 +1,49 @@
-require('../css/admin.scss');
-const $ = require('jQuery');
+import '../scss/admin.scss';
 
 /**
  * fm-overlays-admin.js
  *
- * @created 1/8/16 2:44 PM
  * @author Alley Interactive
  * @package fm-overlays
  * @description JS manipulations to the fm-overlays admin area.
  *
  */
 
+ /**
+  * Set a condition field's wrapper label to the same value as the current option
+  * @param {node} selectEl a <select> element
+  */
+const setLabel = (selectEl) => {
+  const wrapper = selectEl.closest('.fm-fm_overlays_conditionals');
```

Comment:
is selectEl a jquery function? (if so it's not obvious)

Reply:
No, that's just a param for the function, which is a select element. But maybe the naming could be improved.